### DISCUSSION TITLE
Contribute to Flows.

### DISCUSSION THREAD

**Discussion by samay-v** [postmanlabs/postman-flows#151] [Seeking Feedback]
It has been a little over 2 months since we released Postman Flows. It's wonderful to see people show interest and use Flows to solve real-world problems.

We've heard your feedback and iterated over the product while ensuring we keep building towards making Flows more capable. Realigning our plans for Flows with your expectations has been pivotal in making sure we deliver a valuable product.

To make sure that we maintain this alignment, we wanted to know more about you. [This form](https://forms.gle/D3X5EuUJuKECcaSn9) has a few easy questions that help us understand your use-case. This would be valuable to understand your expectation of an efficient API workflow builder.

Within the form you can also choose to participate in more in-depth research sessions. This also opens up your way into the Postman Flow Champions channel where you'd hear about new features first and have access to the full team who is building flows for you.

We'd appreciate it if you took a minute out and filled the survey form.

[https://forms.gle/D3X5EuUJuKECcaSn9](https://forms.gle/D3X5EuUJuKECcaSn9)

**Comment by WilliamWCYoung** [comment]
Hi Samay, looks like I need to be authorized to complete this form? Have the permissions been made public?

**Reply by alexei-moneyr** [reply]
Same here.... no permissions (

**Reply by WilliamWCYoung** [reply]
Works now!

**Reply by samay-v** [reply]
Hey, updated the form access. You should be able to see it now 👍🏽
